scripts/bootstrap.js

```diff
@@ -35,7 +35,7 @@ ls("packages/*").forEach(function (loc) {
 
 var completed = false;
 
-var bar = new ProgressBar(":packagename ╢:bar╟", {
+var bar = !/^win/.test(process.platform) && new ProgressBar(":packagename ╢:bar╟", {
```

Comment:
Sure, if you prefer. Or I can make it depend on [is-windows](https://www.npmjs.com/package/is-windows).